rust_snuba/rust_arroyo/src/processing/mod.rs

```diff
@@ -204,6 +207,8 @@ impl<TPayload: Clone + 'static> StreamProcessor<TPayload> {
                 Some(_) => return Err(RunError::InvalidState),
             }
         };
+
+        let poll_start = Instant::now();
```

Comment:
isnt this just an unused variable now?

Reply:
sorry, put the metric in the wrong place earlier. pushed a fix